### PR TITLE
Fix typos in `cresources.md`

### PR DESCRIPTION
**Pull request by mbercx**

### CODE REVIEW

File: docs/jobs/cresources.md [hunk 1]
```diff
@@ -131,8 +131,8 @@ allocated by HQ.
 
 ## NUMA allocation strategy
 
-Worker automatically detect number of CPUs and on Linux system it also detects partitioning into sockets.
-When NUMA architecture is automatically detected, indexed resource with groups is used for resource "cpus".
+Workers automatically detect the number of CPUs and on Linux systems it also detects their partitioning into sockets.
```
Comment (Kobzol) [inline docs/jobs/cresources.md:134]:
```suggestion
Workers automatically detect the number of CPUs and on Linux systems they also detect their partitioning into sockets.
```

File: docs/jobs/cresources.md [hunk 2]
```diff
@@ -131,8 +131,8 @@ allocated by HQ.
...
+When the NUMA architecture is automatically detected, indexed resource with groups is used for resource "cpus".
```
Comment (Kobzol) [inline docs/jobs/cresources.md:135]:
```suggestion
When a NUMA architecture is automatically detected, indexed resource with groups is used for resource "cpus".
```

**Review by Kobzol — APPROVED** [review]
Two small nits, otherwise LGTM. Thank you!

**Comment by mbercx** [comment]
Agreed on both nits. 👍 Good to go for me!

**Pull request merged**